lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -859,10 +859,6 @@ def on_tls_conf_set(
             if self.tls.is_fully_configured():
                 try:
                     self.tls.reload_tls_certificates()
-                except OpenSearchHttpError:
-                    logger.error("Could not reload TLS certificates via API, will restart.")
-                    self._restart_opensearch_event.emit()
-                else:
```

Comment:
I am not sure I see the value of the `else` here. Moved the code while testing but I kept it as I think it makes everything easier to read

Reply:
I will keep the `else`. I still think it would only matter if there are any other step down in this function that may throw `OpenSearchHttpError`, which is not the case, thou. Otherwise having `else`  or not is irrelevant.

Given this is more a style in this case, I agree leaving things explicit makes more sense.